### PR TITLE
Add bioconda to reverse_proxy

### PR DESCRIPTION
**Pull request by AaronJackson**
Annoyingly I can't get molecule to work locally at the moment, so curious to see what happens in the GitHub Action. May end up being multiple commits to debug, but I'll rebase if that happens.

T E S T 1 2 3

### CODE REVIEW

**Comment by AaronJackson** [comment]
Similar error to what I'm getting...
```
Error connecting: Error while fetching server API version: Not supported URL scheme http+docker
```

**Comment by manics** [comment]
It'll be related to https://github.com/docker/docker-py/issues/3256

Try forcing a newer version of `community.docker` https://github.com/ansible-community/molecule-plugins/issues/256

E.g. https://github.com/manics/jupyterhub-ansiblespawner/pull/317

**Comment by AaronJackson** [comment]
Now we are getting somewhere :)

**Comment by AaronJackson** [comment]
That was much more painful than I wanted it to be

File: reverse_proxy/molecule/default/verify.yml [hunk 1]
```diff
@@ -198,3 +202,40 @@
...
+      eval "$(/root/miniconda3/bin/conda shell.bash hook)"
+      env=$(cat /proc/sys/kernel/random/uuid)
+      conda create -n "$env" -c bioconda bamsnap
+      conda activate "$env"
+      python -c "import bamsnap"
+    failed_when: false
+    changed_when: false
+    register: bioconda_install
+
+  - name: ensure plink2 installation was successful
```
Comment (manics) [inline reverse_proxy/molecule/default/verify.yml:237]:
```suggestion
  - name: ensure package installation was successful
```

File: requirements-dev.txt [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ansible==9.5.1
-ansible-compat==24.5.1
+ansible-compat==24.6.1
 ansible-lint==24.5.0
-molecule==24.2.1
+molecule==24.7.0
 molecule-docker==2.1.0
 yamllint==1.35.1
+requests==2.31.0
```
Comment (manics) [inline requirements-dev.txt:7]: Is this still needed after updating Ansible/molecule and `community.docker`?

File: requirements.yml [hunk 3]
```diff
@@ -4,5 +4,7 @@ collections:
     version: 1.3.0
     type: galaxy
   - name: community.general
-    version: 5.3.0
+    version: 8.6.1
     type: galaxy
+  - name: community.docker
+    version: 3.10.1
```
Comment (manics) [inline requirements.yml:10]:
```suggestion
    version: 3.11.0
```
Might as well install the latest

File: requirements.yml [hunk 4]
```diff
@@ -4,5 +4,7 @@ collections:
     version: 1.3.0
     type: galaxy
   - name: community.general
-    version: 5.3.0
+    version: 8.6.1
```
Comment (manics) [inline requirements.yml:7]:
```suggestion
    version: 9.2.0
```

File: requirements-dev.txt (hunk 2 above)
Comment (AaronJackson) [reply requirements-dev.txt:7]: It is on my laptop 😁

Comment (AaronJackson) [reply requirements-dev.txt:7]: Without it I get:

```
"Error connecting: Error while fetching server API version: Not supported URL scheme http+docker"
```

**Review by manics — APPROVED** [review]

**Pull request merged**